packages/donor/src/components/basic/RadioGroup/RadioGroup.tsx

```diff
@@ -4,34 +4,50 @@ import {
   Radio,
   RadioGroup as MaterialRadioGroup,
 } from "@material-ui/core";
+import { AnalyticsButtonType } from "@zm-blood-components/common";
+import { reportClick } from "../../../Analytics";
 
-export type RadioOption = {
+export type RadioOption<T> = {
   value: any;
   label: string;
   isDisabled?: boolean;
+  key: string;
 };
 
-type RadioGroupProps = {
+type RadioGroupProps<T> = {
```

Comment:
Why do we need the `<T>`?

Reply:
Do we currently need it to be anything but a string?

Reply:
1. If this is not required now, let's not add this complexity (הכנה למזגן)
2. This is not strictly related to this PR, so even if you are interested in this change it should be in a different PR.